cmd/cloud-init-server/main.go

```diff
@@ -229,25 +221,30 @@ func startServer() error {
 		openchami_logger.OpenCHAMILogger(log.Logger),
 	)
 
-	// Client sub-router
-	router_client := chi.NewRouter()
-	initCiClientRouter(router_client, ciHandler, wgInterfaceManager)
-	router.Mount("/cloud-init", router_client)
```

Comment:
Mounting things at `/` instead of `/cloud-init` is a welcome change, but it will be a breaking change to users of the release RPM unless
```
http-request replace-path ^/cloud-init(/.*) \1
```
is added to the cloud-init backend in the HAproxy config.

We can certainly make this change when incorporating this PR into the release repo, but I think it's worth pointing out here for posterity.